service/controller/cloudconfig/master_template.go

```diff
@@ -25,7 +25,7 @@ func (c *CloudConfig) NewMasterTemplate(customObject v1alpha1.KVMConfig, data Ig
 		params.Cluster = customObject.Spec.Cluster
 		// Ingress controller service remains in k8scloudconfig and will be
 		// removed in a later migration.
-		params.DisableIngressControllerService = false
+		params.DisableIngressControllerService = true
```

Comment:
Please remove the comment as well - I assume this will go with a release where nginx becomes optional?

Reply:
@MarcelMue It's disabled for azure and now for kvm, but cleanup cannot be done (and comment is still relevant) since the same template is shared and feature still used for aws legacy.

I just created TODO for the cleanup eventually https://github.com/giantswarm/giantswarm/issues/12412

Does that make sense?

Reply:
Makes sense to me.